cli/flox-rust-sdk/src/models/pkgdb.rs

```diff
@@ -27,15 +27,21 @@ pub struct UpgradeResultJSON {
     pub lockfile: Value,
 }
 
+/// The JSON output of a `pkgdb buildenv` call
+#[derive(Deserialize)]
+pub struct BuildEnvResult {
+    pub store_path: String,
+}
+
 #[derive(Deserialize)]
 pub struct UpgradeResult(pub Vec<String>);
 
 #[derive(Debug, Error)]
 pub enum CallPkgDbError {
     #[error(transparent)]
     PkgDbError(#[from] PkgDbError),
-    #[error("couldn't parse pkgdb error as JSON: {0}")]
-    ParsePkgDbError(String),
+    #[error("couldn't parse pkgdb error in expected JSON format: {0}")]
```

Comment:
```suggestion
    #[error("couldn't parse pkgdb error in expected JSON format")]
```


You already have source in the variant so the error would be printed twice in our printer

Reply:
Ah right